src/BenchmarkDotNet/Running/BenchmarkRunnerClean.cs

```diff
@@ -604,5 +606,17 @@ private static void Cleanup(HashSet<string> artifactsToCleanup)
                 }
             }
         }
+
+        private static string removeWasmPrefix(string line)
```

Comment:
Is it possible to disable the printing of "WASM: " in every output line?

If not, please move this logic to [Measurement.Parse](https://github.com/dotnet/BenchmarkDotNet/blob/0445917bf93059f17cb09e7d48cdb5e27a096c37/src/BenchmarkDotNet/Reports/Measurement.cs#L119)

Reply:
Its not possible to remove with out changes to WASM. Those changes might happen, but I am not sure when. So I will move the logic to Measurement.Parse.

Reply:
@naricc actually this happened a few hours ago: https://github.com/dotnet/runtime/pull/38330 😄 . It should be available shortly in the SDK build from https://github.com/dotnet/installer#installers-and-binaries

I agree with Adam that we should remove this here.